plugins_examples/archive/archive.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"regexp"
+)
+
+func main() {
+	s := bufio.NewScanner(os.Stdin) 
+	for s.Scan() {
+		line := s.Text()
+		if strings.Contains(line, "PRIVMSG") == true {
+			skip := false;
+
+	    		re := regexp.MustCompile("https?\\:\\/\\/[[:^space:]]*")
+	    		match := re.FindString(line)
+
+			if match != "" && match != "https://arch.cccp.io/urls.txt" {
```

Comment:
suggestion: You could add and use `whitelist` and and `blacklist` variables (containing string or even regex)

Reply:
I've updated it to be a file, so it's easier to blacklist things